beacon_node/eth2_libp2p/src/rpc/mod.rs

```diff
@@ -233,9 +233,23 @@ where
                         }))
                 }
                 Err(RateLimitedErr::TooLarge) => {
-                    // we set the batch sizes, so this is a coding/config err
-                    crit!(self.log, "Batch too large to ever be processed";
-                        "protocol" => format!("{}", req.protocol()));
+                    // we set the batch sizes, so this is a coding/config err for most protocols
+                    let protocol = req.protocol();
+                    if matches!(protocol, Protocol::BlocksByRange) {
+                        debug!(self.log, "Blocks by range request will never be processed"; "request" => %req);
+                    } else {
+                        crit!(self.log, "Request size too large to ever be processed"; "protocol" => %protocol);
```

Comment:
I think we don't need this case right? I think its not just a coding error. 

We could hit this if for example someone asked a request with a maximum BlocksByRoot size right? i.e they fill the request with as many roots as the RPC allows, in principle we could hit this then right?

Reply:
the limit for BlocksByRoot is MAX_REQUEST_BLOCKS, and this error is triggered if the weight of the request is bigger than that. This error implies that we are either not allowing requests up to the largest allowed size or that the rpc is letting pass requests that do not comply. So I think this is indeed a CRIT